src/Microsoft.DotNet.Interactive/Microsoft.DotNet.Interactive.csproj

```diff
@@ -39,6 +39,7 @@
     <PackageReference Include="FSharp.Compiler.Private.Scripting">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
Is this needed?

Reply:
without that build fails due to conflict resolution on the package